src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -51,7 +51,9 @@ class InputEditorDecorations extends Disposable {
 
 		this.updateInputEditorDecorations();
 		this._register(this.widget.inputEditor.onDidChangeModelContent(() => this.updateInputEditorDecorations()));
-		this._register(this.widget.onDidChangeParsedInput(() => this.updateInputEditorDecorations()));
+		this._register(this.widget.onDidChangeParsedInput(() => {
```

Comment:
I don't think this is necessarily an improvement, we use the single-line version a lot

Reply:
I think it is a readability improvement in the sense it makes it more explicit, hence take less brain cycles to parse the syntax.

But the main reason for the change is not readability - it takes less keystrokes to change this code in the future, and you can set a breakpoint immediately during debugging.

Do you know what is the main reason for using this pattern in the codebase? Is it mostly less typing or some motives involved?

Reply:
FWIW I use the original style because it's more compact.

Reply:
Reverted.